src/Microsoft.HttpRepl/Commands/ChangeDirectoryCommand.cs

```diff
@@ -57,6 +57,11 @@ protected override Task ExecuteAsync(IShellState shellState, HttpState programSt
                 {
                     IDirectoryStructure s = programState.Structure.TraverseTo(programState.PathSections.Reverse());
 
+                    if (s.RequestInfo == null || s.RequestInfo.Methods.Count == 0)
```

Comment:
a) What do you think about combining this if with the ternary operator on the next line (which has the opposite predicate) into a consolidated if-else?

b) tests?

Reply:
also, nit: use `is null` for null checks.

Reply:
Done, plus some other cleanup in 1332027 and 2549121.